test/Engine/ProtoTestFx/DebugService.cs

```diff
@@ -70,8 +71,10 @@ public bool RunScript(string dsPath)
                 core.Options.RootModulePathName = ProtoCore.Utils.FileUtils.GetFullPathName(dsPath);
                 core.Compilers.Add(ProtoCore.Language.kAssociative, new ProtoAssociative.Compiler(core));
                 core.Compilers.Add(ProtoCore.Language.kImperative, new ProtoImperative.Compiler(core));
-                core.Configurations.Add(Autodesk.DesignScript.Interfaces.ConfigurationKeys.GeometryFactory, GeometryFactoryName);
-                core.Configurations.Add(Autodesk.DesignScript.Interfaces.ConfigurationKeys.PersistentManager, PersistenceManagerName);
+
+                runtimeCore = core.__TempCoreHostForRefactoring;
```

Comment:
@ke-yu yes, Core and RuntimeCore will be full split and completely independent of each other.
__TempCoreHostForRefactorign will be completely removed 

So essentially visualize this:
    Core core = Compile(sourcecode)
    RuntimeCore runtimeCore = new RuntimeCore (core.RuntimeData)
    Mirror = runtimeCore.Execute()